### DISCUSSION TITLE
how to change rewards target addresses on linux cli

### DISCUSSION THREAD

**Discussion by ferdiina** [Chia-Network/chia-blockchain#6544] [Support]
how to change rewards target addresses on linux cli
i  check all chia command  and i can't which command and argument to change target address.
windows gui is easy to do this .

**Comment by blacks1ne** [comment]
If we are speaking of the upcoming pooling model with pool changes, https://www.youtube.com/watch?v=XzSZwxowPzw&t=2046s
According to Mariano S, in the future, the _chia plotnft_ will support pool switching command, for now, it only supports the _create_ and _show_ commands.
Thus, I am not sure the pool switching is possible even via GUI on Windows as in the video only the plotnft creation was shown and the CLI on any OS should be the same unless we speak of different chia app versions.

**Comment by loppefaaret** [comment]
To change the "Farming Target Address" from CLI, you need to manually edit `config.yaml` and edit the 2 entries of `xch_target_address:` to the address you want to use.
I would restart chia services after manually editing `config.yaml`

**Comment by BrandtH22** [comment]
@ferdiina , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
